backend/organization/views/project_views.py

```diff
@@ -23,14 +37,102 @@ def get_serializer_class(self):
         return ProjectMinimalSerializer
 
 
-class ProjectAPIView(ListAPIView):
-    lookup_field = 'pk'
-    serializer_class = ProjectSerializer
-    pagination_class = None
+"""
+Note: I am splitting organization project creation and personal project creation into
+two separate APIs. It makes more sense do two APIs so that we can manage user accessibility
+by project or organization member roles.
+"""
+
+
+class OrganizationProjectsView(APIView):
+    permission_classes = [OrganizationProjectCreationPermission]
+
+    def post(self, request, organization_id):
+        organization = check_organization(organization_id)
+        if not organization:
+            return Response({
+                'message': 'Organization not found'
+            }, status=status.HTTP_404_NOT_FOUND)
+        project_creation_pre_checks(request.data)
+        project = create_new_project(request.data)
+
+        if 'order' in request.data:
+            order = request.data['order']
+            ProjectParents.objects.create(
+                project=project, parent_organization=organization,
+                parent_user=request.user, order=order
+            )
+
```

Comment:
In the "share project" interfaces you can already add your team members. Should it be possible to pass team members to this API and have them be created by this view like the projectparents or should the frontend do an additional request for that? I would vote for the first option. The same thing applies to the personal project create API.

Reply:
Yeah I think with create API we can add project members in the same API. For editing each project we should create a separate API to add project members. Hope that makes sense.